### ISSUE TITLE
A11y: Missing alternative text for Open Graph image

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7105]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The store pages use an Open Graph image for Facebook and other social media where applicable.
However, no associated alternative text with the corresponding og:image:alt element is specified. 
This information is used to enable the display of an alternative text on the target platforms.

*_Expected behaviour:_*

It is recommended to specify an `og:image:alt` meta property declaration where applicable.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to a page using `og:image` meta property (e.g. home page)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40862**



[created from NEXT-40862, comment 541923]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40862, comment 541981]

**Comment by Phil23** [comment]
Hey @blacksheep--, 

I did some research on the `og:image:alt` tag, and there seem to be different opinions on this. In addition, the tag isn't actually used by Facebook itself. If you share a webpage on the platform, the linked image from the `og:image` gets the `og:title` as the `alt` tag set, and the `og:image:alt` is ignored. I verified this myself.

We could do the same within Shopware, but we could just set the title information into the alt tag because we do not have further information from the theme config available, which is what Facebook already does.

Here is an interesting article from another tool, that describes this problem in detail:

https://yoast.com/developer-blog/why-we-dont-set-the-og-image-alt-tag/

If users want to specify the alt tag never less, they can add it via a simple template adjustment of the Twig block `layout_head_meta_tags_opengraph` on their own.

For now, we won't change anything in that regard, due to the described information.
